### PR TITLE
Minor tweaks to apa.csl

### PR DESCRIPTION
**Pull request by bwiernik**
Several minor tweaks to the core APA style. The date for "speech" item is discussed here (https://forums.zotero.org/discussion/45252/apa-6th-conference-paper-error/#Item_9). Other notes are in commit messages.

Do you have a script to automatically update the various APA offshoots?

### CODE REVIEW

**Comment by adam3smith** [comment]
no, I'm too worried that a script would get things wrong. I do that manually with a diff viewer, which is reasonably fast.

File: apa.csl [hunk 1]
```diff
@@ -129,7 +129,7 @@
       <else>
         <choose>
           <if variable="DOI">
-            <text variable="DOI" prefix="doi:"/>
+            <text variable="DOI" prefix="http://dx.doi.org/"/>
```
Comment (adam3smith) [inline apa.csl]: I know they say dx.doi.org, but could we go with doi.org? It's both shorter and preferred by CrossRef

Comment (bwiernik) [inline apa.csl]: I asked the APA Style Experts. They basically said it didn't matter but that the technically correct version would be "dx." I think we can safely just do "doi.org".

File: apa.csl [hunk 2]
```diff
@@ -378,14 +384,21 @@
...
+            <if variable="volume" match="any">
+              <group>
+                <text term="volume" form="short" text-case="capitalize-first" suffix=" "/>
+                <number variable="volume" form="numeric"/>
+              </group>
+            </if>
+            <else>
+              <group>
+                <text term="volume" form="short" plural="true" text-case="capitalize-first" suffix=" "/>
+                <number variable="number-of-volumes" form="numeric" prefix="1-"/>
```
Comment (adam3smith) [inline apa.csl]: it's been there for a long time, but this has got to be an en dash, not a hyphen, right?

Comment (adam3smith) [inline apa.csl]: yup, it should be: http://blog.apastyle.org/apastyle/2013/09/how-to-cite-an-anthology-or-collected-works.html
would you mind fixing this?

File: apa.csl (hunk 1 above)
Comment (adam3smith) [inline apa.csl]: yeah, let's.

File: apa.csl (hunk 2 above)
Comment (bwiernik) [inline apa.csl]: Do I enter it as prefix="1&#8211" or just type the en dash?

**Comment by adam3smith** [comment]
for the en dash - both a literal and an escaped en dash work. We have a preference for escaping for hard to distinguish characters, so what you did is great.

**Comment by adam3smith** [comment]
thanks. 10 commit messages seemed like a bit much, so I added this manually. I copied all relevant commit messages over so we have the history.

**Comment by bwiernik** [comment]
Sorry for the mess.

**Comment by adam3smith** [comment]
no, very helpful to have the details.

**Pull request closed without merging**